src/util/util.js

```diff
@@ -128,13 +167,37 @@ function joinArrayUnique(arr1 = [], arr2 = []) {
   }, []);
 }
 
+/**
```

Comment:
you can remove `joinArrayUnique` because as far as I know it was only used in one place. But double check :)

Reply:
Done